src/ui/components/Toolbar.tsx

```diff
@@ -221,7 +259,7 @@ export default function Toolbar() {
             />
           ) : (
             <ToolbarButton
-              icon="folder"
+              icon="playwright"
```

Comment:
This will show the playwright icon for regular (non-test suite) replays. Maybe we should make a `TestSuitesToolbarButton` that picks between the (eventually) various icons based on the test runner name?

Reply:
I was wrong ... ignore this